code-challenges.test.js

```diff
@@ -50,3 +122,32 @@ const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
 // b) Create the function that makes the test pass.
+
+describe("fullHouse", ()=>{
+    it("takes in an array of 5 numbers and determines whether or not the array is a “full house”. A full house is exactly one pair and one three of a kind.", ()=>{
+        expect(fullHouse(hand1)).toEqual(true)
+        expect(fullHouse(hand2)).toEqual(false)
+        expect(fullHouse(hand3)).toEqual(false)
+        expect(fullHouse(hand4)).toEqual(true)
+    })
+})
+
+//Pseudo code
+//Create a function named fullhouse
+//takes in an array
+//use a for loop
+//iterate over the array to find out if the value is the same across the first three indexes
+//do the same to find out if the value is the same across the last two
+//do that for every possible combination of 5
+//returns a boolean value
+
+const fullHouse = (array) => {
+    let newHand = []
+ for(let i = 0; i < newHand.length; i++) {
+    if (array[i] === array[i-1] && array[i] === array[i +1])
+    return newHand.push(array)
+    else if (array[i] === array[i-2] && array[i] === array[i +2])
+ } return newHand.push(array)
+}
+
+// I tried to work through this one as if it was an interview question. I don't think id' have gotten the job. My blocker is that I don't know how to do this without having a million combinations that I'd have to code for. I feel like I can do everything separately, but when I put them all together my mind goes blank. I need to go back to javascript and work on some more practice problems, I got too soft doing backend lol. 
```

Comment:
You got it! Then hit the million combinations! Better to have working code than and refactor than to not. 

Your pseudo code is on point and you are just missing the goal of returning a boolean in your code other than that your almost solid!